dm/pkg/schema/tracker.go

```diff
@@ -134,14 +139,24 @@ func (tr *Tracker) Init(
 
 	upTracker := schematracker.NewSchemaTracker(lowerCaseTableNames)
 	dsSession := mock.NewContext()
-	dsSession.GetSessionVars().StrictSQLMode = false
+	dsSession.SetValue(ddl.SuppressErrorTooLongKeyKey, true)
 	downTracker := &downstreamTracker{
 		downstreamConn: downstreamConn,
 		se:             dsSession,
 		tableInfos:     make(map[string]*DownstreamTableInfo),
 	}
 	// TODO: need to use upstream timezone to correctly check literal is in [1970, 2038]
-	se := executorContext{Context: mock.NewContext()}
+	sctx := mock.NewContext()
+	store, err := mockstore.NewMockStore()
```

Comment:
DM should not use mock store, it's a regression. why do we add it?

Reply:
I thought that, when bump the tidb on the master branch, meet some case failed caused by the store is panic, the set it to the mock store.

Reply:
we need to fix them at tidb side, like https://github.com/pingcap/tidb/pull/50939

Reply:
I am curious why do you think this is a regression? is there any negative impact by using mock store ?

Reply:
> I am curious why do you think this is a regression? is there any negative impact by using mock store ?

https://github.com/pingcap/tiflow/issues/5334